docs/man/git-lfs-lock.1.ronn

```diff
@@ -0,0 +1,27 @@
+git-lfs-lock(1) -- Set a file as "locked" on the Git LFS server
+===============================================================
+
+## SYNOPSIS
+
+`git lfs lock` [options] <path>
+
+## DESCRIPTION
+
+Sets the given file path as "locked" against the Git LFS server, with the
+intention of blocking attempts by other users to update the given path. Locking
+a file requires the file to exist in the working copy.
+
+## OPTIONS
+
+* `-r` <name> `--remote=`<name>:
+  Specify the Git LFS server to use. Ignored if the `lfs.url` config key is set.
+
+* `--json`:
```

Comment:
A caveat of this flag is that `Error()` calls don't format their output as JSON, mostly as a function of us not having a standardized way to report errors back out to external programs.

I think it would be good to note here that any un-parseable JSON (+ a non-zero exit code) should be treated as the error message.